go.mod

```diff
@@ -48,45 +45,38 @@ require (
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
```

Comment:
there is a comment above "// pinning to specific versions is required because we are consuming
// the k8s.io/kubernetes module from the tests (via the "k8s.io/kubernetes/test/e2e/framework" package)
// See https://github.com/kubernetes/kubernetes/issues/90358#issuecomment-617859364"

Was this not supposed to delete this require block?

Reply:
removed